FrendsTaskTemplate/README.md

```
@@ -0,0 +1,94 @@
+# Frends Task template
+
+[![BuildMaster](https://github.com/FrendsPlatform/FrendsTasks/actions/workflows/template_task_release.yml/badge.svg)](https://github.com/FrendsPlatform/FrendsTasks/actions/workflows/template_task_release.yml)
+[![License: MIT](https://img.shields.io/badge/License-MIT-green.svg)](https://opensource.org/licenses/MIT)
+
+This template can be used to create new .NET (previously .NET Core) Tasks for [Frends](https://frends.com) integration
+platform. This enables you to start writing code without any hassle with project formats etc.
+
+You can learn more about custom Tasks [here](https://docs.frends.com/en/articles/2206746-custom-tasks).
```

markdownlint-cli2 (0.22.0)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)